modules/Kconfig.cmsis_dsp

```diff
@@ -770,6 +13,20 @@ config CMSIS_DSP_NEON
 	  This option enables the NEON Advanced SIMD instruction set, which is
 	  available on most Cortex-A and some Cortex-R processors.
 
+config CMSIS_DSP_NEON_EXPERIMENTAL
+	bool "Neon Instruction Set"
+	depends on CPU_CORTEX_A
+	help
+	  This option enables the NEON Advanced SIMD instruction set, which is
+	  available on most Cortex-A and some Cortex-R processors.
```

Comment:
Why do we have both `CMSIS_DSP_NEON` and `CMSIS_DSP_NEON_EXPERIMENTAL`?

Reply:
In case if someone wants to enable the neon experimental....? This is an option defined within CMSIS-DSP

Reply:
If there is a Kconfig option for it, then it should probably be OK. I guess some additional cflags are necessary to enable the relevant builtins? I guess that is potentially @stephanosio's main concern - do we need additional toolchain profiles in the SDK?